src/gc/gc.cpp

```diff
@@ -11643,7 +11644,10 @@ void gc_heap::adjust_limit_clr (uint8_t* start, size_t limit_size, size_t size,
     }
 
     // verifying the memory is completely cleared.
-    //verify_mem_cleared (start - plug_skew, limit_size);
+    //if (!(flags & GC_ALLOC_ZEROING_OPTIONAL))
+    //{
+    //    verify_mem_cleared(start - plug_skew, limit_size);
+    //}
```

Comment:
Why is both the old and new code commented out?

Reply:
I think this is too slow even for debug. 
I used it for investigations and had to adjust for `GC_ALLOC_ZEROING_OPTIONAL`, but decided to keep the whole thing commented as it was before.